.stylelintrc.yaml

```diff
@@ -0,0 +1,101 @@
+---
+
+extends: stylelint-config-standard
+
+rules:
+  linebreaks: unix
+  indentation: tab
+  unicode-bom: never
+
+  max-empty-lines: 2
+  max-line-length: 80
+  max-nesting-depth: 2
+  no-eol-whitespace: true
+  no-empty-first-line: true
+  no-missing-end-of-source-newline: true
+
+  no-unknown-animations: true
+
+  number-leading-zero: never
```

Comment:
:'(

Reply:
I think most of our styling looks better without a leading zero. Do you feel strongly on this?